packages/firestore/rollup.config.es2017.js

```diff
@@ -45,6 +48,11 @@ import {
 // The in-memory builds are roughly 130 KB smaller, but throw an exception
 // for calls to `enablePersistence()` or `clearPersistence()`.
 //
+// We use two different rollup pipelines to take advantage of tree shaking,
+// which Rollup only supports in pure ES2017 builds. The build pipeline in this
```

Comment:
This is not accurate. rollup can tree shake ES5 builds as long as they are in the ESM format. The limitation is specific to the Typescript classes compiled down to ES5. 

Also do you mean `post-tree shaken ES2017` builds? IIUC, treeshaking is done in this phase, the second phase just changes the module format.

Reply:
Updated the comment. I did mean "pre tree-shaken", as in "these builds are already tree shaken", which is the same as "post tree-shaken".  Hence, I dropped the term.